lib/temporal/testing/local_workflow_context.rb

```diff
@@ -185,6 +185,10 @@ def on_signal(&block)
         raise NotImplementedError, 'Signals are not available when Temporal::Testing.local! is on'
       end
 
+      def on_query(&block)
```

Comment:
Why was this removed? There's no actual support for local query so this exception should remain.

Reply:
Nothing was removed here, this was added. I will need to fix the method signature on it, though